python/src/gen_gherkin/gen_gherkin_helpers.py

```diff
@@ -1,5 +1,6 @@
 """Helper methods for extracting and sorting pertinent data for use in generating feature files."""
 from aac.context.definition import Definition
+import re
```

Comment:
```suggestion
from re import sub

from aac.context.definition import Definition
```

Remember to keep your imports organized, external first then internals. And remember to try to stick to using `from x import y` to keep from ballooning the install.